modules/scheduler/batch-job-template/variables.tf

```diff
@@ -91,10 +91,12 @@ variable "log_policy" {
   }
 }
 
-variable "runnable" {
-  description = "A string to be executed as the main workload of the Google Cloud Batch job. This will be used to populate the generated template."
-  type        = string
-  default     = "## Add your workload here"
+variable "runnables" {
+  description = "A list of strings to be executed as a sequence of shell scripts as the main workload of the Google Cloud Batch job. These will be used to populate the generated template."
+  type        = list(string)
```

Comment:
In previous discussion we had maybe using `list(object(...))`.

This would allow us to specify additional parameters. In our notes we had:
```
- id: step 1
  script: echo 'hello world'
  execution_mode: ?
```

We could leave off execution_mode for now since we don't have that behavior settled, but it would leave the door open to it in the future without breaking changes again. WDYT?

Whatever we agree on, I can help to hone the tf code if you need help.

Reply:
Sure, that will be more flexible for future enhancements. I have updated it to type(object) and updated the default value and example files as well.

Reply:
Is there anything useful that we can/need to do with id right now? Like can we put it somewhere in the job such that it would show up in pantheon or in logging. If it will be unused then I would remove it, but still go with the dict style `script: <string>`, to give us room to grow.